ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-rollback-stable.yaml

```diff
@@ -99,4 +99,4 @@ zz_generated_metadata:
   branch: release-4.16
   org: openshift
   repo: openshift-tests-private
-  variant: amd64-rollback-ec
```

Comment:
Also need update the line like https://github.com/openshift/release/pull/52084/commits/421132164f2872ebfa66b7175cb53d1a94503d24#diff-8de710535f0a5755e0356538df0e71e1d94eb52274b6896749e96741bb5aff1cL71, right?

Reply:
@jianlinliu yes, good catch. Thanks